sunkit_image/coalignment/decorators.py

```diff
@@ -0,0 +1,21 @@
+__all__ = ["register_coalignment_method", "REGISTERED_METHODS"]
+
+# Global Dictionary to store the registered methods and their names
+REGISTERED_METHODS = {}
+
+
+def register_coalignment_method(name):
+    """
+    Registers a coalignment method to be used by the coalignment interface.
+
+    Parameters
+    ----------
+    name : str
+        The name of the coalignment method.
+    """
+
+    def decorator(func):
+        REGISTERED_METHODS[name] = func
+        return func
+
+    return decorator
```

Comment:
It's silly and unnecessarily complicated to have this as a separate file.  Just move this all to `interface.py`.